magento/magento2
# Adding the company and VAT Number values at OPC address.

### Description (*)


Adding the company and VAT Number values at OPC address.

### Related Pull Requests
https://github.com/magento/partners-magento2b2b/pull/538


### Fixed Issues (if relevant)

N/A

### Manual testing scenarios (*)

   1. Enable showing company and VAT Number data at the front
   2. Check if it is available at OPC.

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [ ] resolves magento/magento2#31439: Region data fix

### Resolved issues:
1. [x] resolves magento/magento2#31652: Adding the company and VAT Number values at OPC address.
